tmt-app/src/Components/Stepper.jsx

```diff
@@ -0,0 +1,96 @@
+import * as React from "react";
+import { useLocation } from "react-router-dom";
+import { CustomTheme } from "../Styling/CustomStyling";
+import HomeIcon from "../Images/icons/boba_progress/Home_Current.png";
+import SubjectsIcon from "../Images/icons/boba_progress/Subjects_Current.png";
+import SoftSkillsIcon from "../Images/icons/boba_progress/SoftSkills_Current.png";
+import OrderUpIcon from "../Images/icons/boba_progress/OrderUp_Current.png";
```

Comment:
![image](https://github.com/BobaTalks/pick-a-major/assets/12107969/cc386f0d-817b-44cd-9666-02dae3184d1f)
The images like this are coming out blurry. Were we not able to just acquire the boba icon as an svg and implement the text and spacing ourselves?

Reply:
Added in the new logos.